2d/include/pcl/2d/impl/kernel.hpp

```diff
@@ -131,9 +130,10 @@ pcl::kernel<PointT>::loGKernel(pcl::PointCloud<PointT>& kernel)
 
   for (int i = 0; i < kernel_size_; i++) {
     for (int j = 0; j < kernel_size_; j++) {
+      float temp = 0;
       int iks = (i - kernel_size_ / 2);
       int jks = (j - kernel_size_ / 2);
-      temp = float(double(iks * iks + jks * jks) / sigma_sqr);
+      float temp = float(double(iks * iks + jks * jks) / sigma_sqr);
```

Comment:
Why did you add L133?

Reply:
My mistake. The [Suggested change](https://github.com/PointCloudLibrary/pcl/pull/3857#discussion_r403250720) didn't remove the line, and I missed that.